app/views/pageflow/entries/_navigation.html.erb

```diff
@@ -7,11 +7,39 @@
 </div>
 
 <script>
+pageflow.Invert = {
```

Comment:
I would like a one-line description of the class, why this class was created (what does it do).